### PR TITLE
chore: move contrib to top nav

### PR DESCRIPTION
**Pull request by ethangj**
Initial changes to the pages, ordering, menus and structure.

### CODE REVIEW

**Comment by hferentschik** [comment]
nitpick, the commits should really be squashed, if not into a single commit, then at least into two. In hindsight, it is not relevant that it took 4 commits to sort out broken links.

**Comment by ethangj** [comment]
@hferentschik I actually went back and forth on ditching the dropdown, just thought some people might object. Went ahead and removed it, I agree its better.

Disagree on the wording though, it's a call to action vs a normal nav item. I don't really love the wording, but it needs to highlight that it's actions you can take vs content to read.

**Comment by hferentschik** [comment]
> @hferentschik I actually went back and forth on ditching the dropdown, just thought some people might object. Went ahead and removed it, I agree its better.

+1 Soooooo much better. 

> Disagree on the wording though, it's a call to action vs a normal nav item. I don't really love the wording, but it needs to highlight that it's actions you can take vs content to read.

Well, the action is maybe on the landing page, but most of the content now is "reading" material. All the stuff on how to contribute code and documentation is basically just "documentation". IMO "Community" is better. That said, I won't fight over it. Removing the dropdown was the actual issue.

There is one more thing now, the new menu item "Join & Contribute" is losing context. You select it and it should stay selected (bold) while you are browsing the corresponding section of the site. This is how the other menu items work. It does not work with "Join & Contribute". If I'd take a guess it might be the '&' or the ' ' (spaces) around it.

**Comment by ethangj** [comment]
@hferentschik Changed the title and fixed the active state issue. As for the squashing, need you to pair with me and help get that right.

File: content/en/community/office_hours/_index.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 title: Office Hours
 linktitle: Office Hours
 type: docs
-weight: 30
+weight: 9
```
[the remark this answers is not in the document]
Comment (ethangj) [reply content/en/community/office_hours/_index.md:5]: That's smart! I wish I had thought of that for many past projects 😂

**Comment by ethangj** [comment]
@hferentschik all set

**Comment by hferentschik** [comment]
> @hferentschik all set

very nice indeed :-)

**Pull request merged**